### DISCUSSION TITLE
Read-Only Mode / View-Only Mode

### DISCUSSION THREAD

**Discussion by Coding-Sock** [kwsch/PKHeX#3754] [Ideas]
The idea behind this is to have an option that disables the ability to make modifications in order to accident proof the app for those who are just using it to view data.

It could be enabled or disabled from the settings, and when it has been enabled it would disable the buttons that set and save changes and/or the save and set functionality.

This feature would be for those who are only interested in using the app to view the data but are not interested in making any modifications. This way they would not need to worry about about any miss-clicks, hitting the wrong key shortcuts, or the need to restore backups to remove accidental modifications.

**Comment by kwsch** [comment]
If you're not interested in making any modifications, just don't export the save file or set it back to a slot. 
If you accidentally write a slot, you  can use the Undo feature.

It would be quite a bit of work to make the GUI readonly, tons of things would need their shortcuts kneecapped.

**Comment by Bappsack** [comment]
You have to export in order to save changes. 

So just don't export?
